aptos-move/framework/aptos-framework/sources/reconfiguration.move

```diff
@@ -115,11 +120,17 @@ module aptos_framework::reconfiguration {
             return
         };
 
+        assert!(current_time > config_ref.last_reconfiguration_time, error::invalid_state(EINVALID_BLOCK_TIME));
+        let epoch_duration = if (config_ref.last_reconfiguration_time == 0) {
```

Comment:
Can you document that this is necessary or otherwise the validators would receive massive rewards in the network's first block? Are there other reasons I missed?